definitions/bat_register_test.js

```diff
@@ -0,0 +1,966 @@
+/* For use by dfe-analytics-dataform developers working with Register test data only - for example code to use in your project, see definitions/example.js */
+
+const dfeAnalyticsDataform = require("../");
+

```

Comment:
I'm not sure what is happening in this file - is this to provide some test data for missing deletions work?